en/faq.md

```diff
@@ -128,3 +128,9 @@ To ensure persistent storage of data, TiDB clusters deployed by TiDB Operator us
 To ensure data safety in case one node is down, PD and TiKV use [Raft Consistency Algorithm](https://raft.github.io/) to replicate the stored data as multiple replicas across nodes.
 
 In the bottom layer, TiKV replicates data using the log replication and State Machine model. For write requests, data is written to the Leader node first, and then the Leader node replicates the command to its Follower nodes as a log. When most of the Follower nodes in the cluster receive this log from the Leader node, the log is committed and the State Machine changes accordingly.
+
+## Is the cluster unavailable when the Ready field of TiDBCluster is false?
+
+After execute `kubectl get tc` command, maybe you notice that the Ready field of TiDBCluster false.
+
+It doesn't mean that the TiDB cluster is unavailable, maybe cluster is upgrading or scaling. You can check whether cluster is unavailable by connecting TiDB.
```

Comment:
```suggestion
After you execute the `kubectl get tc` command, if the output shows that the **Ready** field of a TiDBCluster is false, it does not mean that the corresponding TiDBCluster is not available, because the cluster might be in the process of upgrading or scaling up.
	
To check whether a TiDBCluster is unavailable, you can try connecting to TiDB.
```